backend/layers/common/validation.py

```diff
@@ -22,7 +22,11 @@ def _verify_collection_metadata_fields(
 
     def check(key):
         value = getattr(metadata, key)
-        if check_existence and not value:
+        if check_existence and value is None:
+            # if checks_existence is true, value cannot be None since it must be required
+            errors.append({"name": key, "reason": "Cannot be blank."})
```

Comment:
lets give this error reason a different value than the one in the next block (or vice-versa). We should distinguish the error paths